KeyCloak SSO How to Guide (#1477)
kestra-io/docs: content/docs/06.enterprise/05.sso.md

```

With these settings, Kestra is now configured to use OIDC for SSO with your chosen providers. Ensure that all placeholders are replaced with actual values obtained from the provider's configuration process.

## Using KeyCloak as an OICD SSO Provider

In your KeyCloak realm be to sure to have a client with "Client Authentication" property set to "On" and "Standard flow" checked.

> In the "Valid redirect URI" property make sure to have `https://{{ yourKestraServerURL }}/oauth/callback/keycloak`. In "Valid post logout redirect URIs" property make sure to have to have `https://{{ yourKestraServerURL }}/logout`.

Reviewer: ```suggestion
Set the "Valid redirect URI" property to `https://{{ yourKestraServerURL }}/oauth/callback/keycloak`. 

Then, set the "Valid post logout redirect URIs" property to `https://{{ yourKestraServerURL }}/logout`.
```
